src/drt/src/db/gcObj/gcNet.h

```diff
@@ -50,7 +50,9 @@ class gcNet : public gcBlockObject
         pins_(numLayers),
         owner_(nullptr),
         taperedRects(numLayers),
-        nonTaperedRects(numLayers)
+        nonTaperedRects(numLayers),
+        isBlockage_(false),
```

Comment:
What does it mean for a net to be a blockage?

Reply:
gcNet as you may know is not necessarily a net. It is building block of all objects in the DRC engine database. So a blockage gcNet is simply a blockage. isBlockage should mark this. However, after some thinking, I think this could be inferred directly from the owner. So I am gonna take another look at this.